server/src/db/seeds.sql

```diff
@@ -6,14 +6,13 @@ INSERT INTO Users (username, email, passwordHash, role) VALUES
-(3, 'Naruto', 'Guided by the spirit demon within him, orphaned Naruto learns to harness his powers as a ninja in this anime adventure series.', 'https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5da0c85bd2c9c10009370984/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS', 'https://jbarteluce.net/files/wp9562833.jpg.webp', TRUE),
-(1, 'SpongeBob Schwammkopf', 'A square yellow sponge named SpongeBob SquarePants lives in a pineapple with his pet snail, Gary, in the city of Bikini Bottom on the floor of the Pacific Ocean. He works as a fry cook at the Krusty Krab. During his time off, SpongeBob has a knack for attracting trouble with his starfish best friend, Patrick. Arrogant octopus Squidward Tentacles, SpongeBobs neighbor, dislikes SpongeBob because of his childlike behavior.', 'https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5d00e8adaab96b5635b2a005/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS', 'https://jbarteluce.net/files/SpongeBob_Schwammkopf.webp', TRUE),
-(2, 'KGW8', 'Local News and Information for Portland, Oregon and surrounding areas.', 'https://lnc-kgw.tubi.video/live.m3u8', 'https://jbarteluce.net/files/kgw8.png', TRUE),
-(3, 'Classic Movies', 'Turner Classic Movies is the source for all things classic films, including clips from yesterday''s Hollywood stars and everyone''s favorite "old movies" whether they''re in black and white or color!', 'https://turnerlive.warnermediacdn.com/hls/live/2023187/tcmwest/noslate/VIDEO_1_5128000.m3u8', 'https://jbarteluce.net/files/TCM_2021.svg.png', TRUE);
+(1, 'Server Host Stream', 'This is the server host stream. It has the streamout.m3u8 url, so the server host can stream their zany antics to the masses.', 'https://localhost:8000/stream/streamout.m3u8', 'https://images.pexels.com/photos/169647/pexels-photo-169647.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2', FALSE);
 
 -- Insert messages
 INSERT INTO Messages (userId, streamId, body) VALUES
 (2, 1, 'Hello, world!'),
 (2, 1, 'This is a test message.');
+
+-- Sample federation member
+--INSERT INTO Federation (fedPublicId, apiUrl, apiPort) VALUES
+--(2, '10.0.0.198', 8000, 2);
```

Comment:
```suggestion
-- Sample federation member
-- INSERT INTO Federation (fedPublicId, apiUrl, apiPort) VALUES
-- (2, '10.0.0.198', 8000, 2);
```

MySQL complained again, this time due to the lack of space after the comment delimiter.